R/navbar.R

```diff
@@ -246,7 +219,13 @@ bs4_navbar_links_tags <- function(links, depth = 0L) {
     }
 
     if (!is.null(x$text) && grepl("^\\s*-{3,}\\s*$", x$text)) {
+
+      if (index == 1) {
```

Comment:
this is the important change.

Reply:
I'd use an `else` block just to make it clear that there are two equal-weight paths.